### PR TITLE
watch your profanity

### PR DESCRIPTION
**Pull request by thijstakken**

### CODE REVIEW

**Comment by lukasvdberk** [comment]
Ik zeg mergen

File: server.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import json, os, subprocess, socket
 from resources.misc import sysinfo
+#from socket import * #socket, AF_INET, SOCK_STREAM
 from resources.protection import proc_check, fake_mutex_code
 from requests import get
 
 port = 64780
-connection = False
+#host_IP = "127.0.0.1"  # need to be loopback
+connection = "nope"
```
Comment (lukasvdberk) [inline server.py:9]: bad practice

**Comment by sonofice** [comment]
Merge runs on old version. Not good. suck cow tit. Please fix first

File: server.py [hunk 2]
```diff
@@ -13,57 +15,79 @@
...
+                except Exception:
+                    result = "command not found"
         else:
-            pass
-            # s.close()
-            # print("command not found")
+            s.close()
+            print("command not found")
 
         conn.sendall(bytes(result, encoding="utf-8"))
```
Comment (lukasvdberk) [inline server.py:93]: what about other characters?

**Review by lukasvdberk — CHANGES_REQUESTED** [review]
mwah

**Pull request closed without merging**